src/services/clusters/clusters.ts

```diff
@@ -276,3 +276,15 @@ export async function updateClusterTemplateSchemaTags(clusterID: number, data: A
     data,
   });
 }
+
+export async function addFavorite(clusterID: number) {
```

Comment:
why not use UpdateCluster ?

Reply:
UpdateCluster will update every field in a cluster, that's unnecessary.